docs/dptp-triage-sop/openshift-priv.md

```diff
@@ -0,0 +1,45 @@
+# `openshift-priv` Organization
+
+## Unrelated histories
+
+A force-push in one of the repositories which has a counterpart in
+`openshfit-priv` will cause the job to fail to synchronize the repositories,
+e.g.:
+
+```
+time="2022-11-01T08:23:40Z" level=warning msg="error occurred while fetching remote and merge" branch=release-4.13 destination=openshift-priv/cluster-api-provider-ibmcloud@release-4.13 error="[failed to merge openshift-cluster-api-provider-ibmcloud/release-4.13: failed with 128 exit-code: fatal: refusing to merge unrelated histories\n, failed to perform merge --abort: failed with 128 exit-code: fatal: There is no merge to abort (MERGE_HEAD missing).\n]" local-repo=/tmp/1970879496/openshift/cluster-api-provider-ibmcloud org=openshift repo=cluster-api-provider-ibmcloud source=openshift/cluster-api-provider-ibmcloud@release-4.13 source-file=openshift-cluster-api-provider-ibmcloud-release-4.13.yaml variant=
+```
+https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-openshift-release-private-org-sync/1587355868737835008
+
+Because merge conflicts are [ignored][private_org_sync_readme] by
+`private-org-sync`, this will not cause the synchronization job to fail;
+instead, the two repositories will silently diverge.  Eventually, this will
+result in support requests when the owners notice their repository has not been
+updated.
+
+Manual intervention is required to reconcile the two repositories.  For example,
+to completely overwrite the private version:
+
+```console
+$ git clone --quiet --origin public https://github.com/openshift/cluster-api-provider-ibmcloud.git
+$ cd cluster-api-provider-ibmcloud/
+$ git remote set-url --push public '' # for safety
+$ git remote add private https://openshift-merge-robot@github.com/openshift-priv/cluster-api-provider-ibmcloud.git
+$ git fetch --quiet --all
+$ git log --oneline -n 1 private/release-4.13
+3bde969f (private/release-4.13) Merge pull request #21 from lobziik/mao-update-f76a8f3a
+$ git log --oneline -n 1 public/release-4.13
+5e3a2bae (public/release-4.13) Merge pull request #38 from Karthik-K-N/merge-09-10-22
+$ git push --force private public/release-4.13:release-4.13
```

Comment:
> ```console
> $ git push --force private public/release-4.13:release-4.13
> ```

What about CVE patches then?

Reply:
That's why this is preceded by a sentence which begins with "for example".

Reply:
But to answer the question more directly, you would do nothing, leaving the two repositories as they are.